### PR TITLE
Adjusted guidance around cluster maximums

### PR DESCRIPTION
**Pull request by ahardin-rh**
Relates to https://issues.redhat.com/browse/PERFSCALE-868

Preview build: https://deploy-preview-32936--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/planning-your-environment-according-to-object-maximums?utm_source=github&utm_campaign=bot_dp

### CODE REVIEW

**Review by mffiedler — APPROVED** [review]
Lgtm

File: modules/openshift-cluster-maximums.adoc [hunk 1]
```diff
@@ -5,97 +5,77 @@
 [id="cluster-maximums_{context}"]
 = {product-title} tested cluster maximums
 
-[options="header",cols="7*"]
+The values in the following table were tested independently of each other and represent the maximum for that particular resource type. It might not be valid to consider these maximums in combinations. Appropriate capacity planning and testing should be performed for large environments.
```
Comment (chaitanyaenr) [inline modules/openshift-cluster-maximums.adoc:8]: We might want to mention that there are other objects created during some the test runs but not close to the maximums. For example, when we test namespaces per cluster, deployments per namespace or number of nodes, we create thousands of pods, services, deployments, secrets, configmaps, builds etc. but are not close to the cluster maximums. Thoughts?

Comment (ahardin-rh) [reply modules/openshift-cluster-maximums.adoc]: @chaitanyaenr Sounds good to me! Updated.

File: scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc [hunk 2]
```diff
@@ -7,13 +7,10 @@ toc::[]
 
 Consider the following tested object maximums when you plan your {product-title} cluster.
 
-These guidelines are based on the largest possible cluster. For smaller
-clusters, the maximums are lower. There are many factors that
-influence the stated thresholds, including the etcd version or storage data
-format.
+These guidelines are based on the largest possible cluster. For smaller clusters, the maximums are lower. There are many factors that
+influence the stated thresholds, including the etcd version or storage data format.
```
Comment (jeana-redhat) [inline scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc:11]: did you intend to unwrap this line?

**Review by jeana-redhat — APPROVED** [review]
Nice consolidation :) One small Q from me, but looks good!

File: scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc (hunk 2 above)
Comment (ahardin-rh) [reply scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc:11]: yes, but looks like there's an extra space that's throwing off the flow; fixing that now! Thanks!

**Pull request merged**